app/src/main/java/com/infomaniak/drive/ui/bottomSheetDialogs/FileInfoActionsBottomSheetDialog.kt

```diff
@@ -95,6 +96,8 @@ class FileInfoActionsBottomSheetDialog : BottomSheetDialogFragment(), FileInfoAc
                 isSharedWithMe = navigationArgs.userDrive.sharedWithMe,
             )
             updateCurrentFile(currentFile)
+
+            binding.fileInfoActionsView.setPrintVisibility(isGone = currentFile.extensionType != ExtensionType.PDF.value)
```

Comment:
Maybe we could add an extension `isPdf` in File.kt to avoid duplicating the logic here. I'm not totally sure it's worth it though.

But we could use the `getFileType()` function to directly check on ExtensionType.PDF without the value.